src/components/NewMessageForm/NewMessageForm.vue

```diff
@@ -20,14 +20,31 @@
 -->
 
 <template>
-	<div class="wrapper">
-		<div class="new-message">
-			<form class="new-message-form">
-				<button class="new-message-form__button icon-clip-add-file" />
-				<button class="new-message-form__button icon-emoji-smile" />
-				<AdvancedInput v-model="text" @submit="handleSubmit" />
-				<button class="new-message-form__button icon-bell-outline" />
-				<button type="submit" class="new-message-form__button icon-folder" @click.prevent="handleSubmit" />
+	<div
+		class="wrapper">
+		<div
+			class="new-message">
+			<form
+				class="new-message-form">
+				<button
```

Comment:
Btw, they should use Actions for this (but not on this pr)

Reply:
Ok